packaging/docker/Dockerfile

```diff
@@ -0,0 +1,194 @@
+    sysstat-10.1.5-19.el7 \
+    tar-1.26-35.el7 \
+    tcpdump-4.9.2-4.el7_7.1 \
+    telnet-0.17-66.el7 \
+    traceroute-2.0.22-2.el7 \
+    unzip-6.0-22.el7_9 \
+    vim-enhanced-7.4.629-8.el7_9 && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+
+WORKDIR /tmp
+
+RUN curl -Ls https://github.com/krallin/tini/releases/download/v0.19.0/tini-amd64 -o tini  && \
+    echo "93dcc18adc78c65a028a84799ecf8ad40c936fdfc5f2a57b1acda5a8117fa82c  tini" > tini-amd64.sha256sum && \
+    sha256sum -c tini-amd64.sha256sum && \
+    chmod +x tini && \
+    mv tini /usr/bin/ && \
+    rm -rf /tmp/*
+
+WORKDIR /
+
+FROM golang:1.16.7-bullseye AS go-build
+
+COPY fdbkubernetesmonitor/ /fdbkubernetesmonitor
+WORKDIR /fdbkubernetesmonitor
+RUN go build -o /fdb-kubernetes-monitor ./...
+
+FROM base as foundationdb-base
+
+WORKDIR /tmp
+ARG FDB_VERSION=6.3.22
+ARG FDB_LIBRARY_VERSIONS="${FDB_VERSION}"
+ARG FDB_WEBSITE=https://www.foundationdb.org
+
+RUN mkdir -p /var/fdb/{logs,tmp,lib} && \
+    mkdir -p /usr/lib/fdb/multiversion && \
+    echo ${FDB_VERSION} > /var/fdb/version
+
+# Set up a non-root user
+RUN groupadd --gid 4059 fdb && \
+    useradd --gid 4059 --uid 4059 --no-create-home --shell /bin/bash fdb && \
+    chown -R fdb:fdb /var/fdb
+
+COPY website /tmp/website/
+
+# Install FoundationDB Binaries
+RUN curl $FDB_WEBSITE/downloads/$FDB_VERSION/linux/fdb_$FDB_VERSION.tar.gz | tar zxf - --strip-components=1 && \
+    for file in fdbbackup fdbcli fdbdr fdbmonitor fdbrestore fdbserver backup_agent dr_agent fastrestore_tool; do \
+        chmod u+x $file; \
+        mv $file /usr/bin; \
+    done
+
+# Install additional FoundationDB Client Libraries
+RUN for version in $FDB_LIBRARY_VERSIONS; do \
+    curl $FDB_WEBSITE/downloads/$version/linux/libfdb_c_$version.so -o /usr/lib/fdb/multiversion/libfdb_c_${version%.*}.so; \
+    done
+
+# Install additional FoundationDB Client Libraries (for sidecar)
+RUN mkdir -p /var/fdb/lib && \
+    for version in $FDB_LIBRARY_VERSIONS; do \
+    curl $FDB_WEBSITE/downloads/$version/linux/libfdb_c_$version.so -o /var/fdb/lib/libfdb_c_${version%.*}.so; \
+    done
+
+RUN curl $FDB_WEBSITE/downloads/$FDB_VERSION/linux/libfdb_c_$FDB_VERSION.so -o /usr/lib/libfdb_c.so
+
+RUN rm -rf /tmp/*
+WORKDIR /
+
+FROM foundationdb-base as fdb-kubernetes-monitor
+
+# Install the kubernetes monitor binary
+COPY --from=go-build /fdb-kubernetes-monitor /usr/bin/
+
+# Runtime Configuration Options
+USER fdb
+WORKDIR /var/fdb
+VOLUME /var/fdb/data
+ENTRYPOINT ["/usr/bin/fdb-kubernetes-monitor"]
+
+FROM foundationdb-base as foundationdb-kubernetes-sidecar
+
+RUN yum -y install \
+    rh-python38-2.0-4.el7 \
+    yum clean all && \
+    rm -rf /var/cache/yum && \
+    source /opt/rh/rh-python38/enable && \
+    pip3 install watchdog==0.9.0
+
+WORKDIR /
+ADD entrypoint.bash sidecar.py /
+RUN chmod a+x /entrypoint.bash /sidecar.py
+USER fdb
+VOLUME /var/input-files
+VOLUME /var/output-files
+ENV ADDITIONAL_ENV_FILE "/opt/rh/rh-python38/enable"
```

Comment:
Why is this needed?

Reply:
to enable python38 in a centos7 image.

Reply:
Ack. thanks.

Reply:
good call, just pushed a change for this.